packages/ember-views/lib/compat/attrs-proxy.js

```diff
@@ -67,50 +37,11 @@ let AttrsProxyMixin = {
     val.update(value);
   },
 
-  _propagateAttrsToThis() {
-    let attrs = this.attrs;
-
-    for (let prop in attrs) {
-      if (prop !== 'attrs' && !this.constructor.__avoidPropagating[prop]) {
-        this.set(prop, this.getAttr(prop));
-      }
-    }
-  },
-
-  initializeShape: on('init', function() {
-    this._isDispatchingAttrs = false;
-  }),
-
-  _internalDidReceiveAttrs() {
-    this._super();
+  propagateAttrsToThis(attrs) {
```

Comment:
Fixed in https://github.com/emberjs/ember.js/commit/6c5f87d61d1893654a1f28f0b25902c1c782311b